views/events.handlebars

```diff
@@ -0,0 +1,66 @@
+    <h2 class="text-center">
+      Current events
+    </h2>
+
+    <section class="d-flex justify-content-around flex-wrap p-2">
+    <div class="card mb-3" style="max-width: 540px;">
```

Comment:
Good for now, but when we have seeds should be updated as a loop using something like: `{{ for each }}`